demisto_sdk/commands/common/hook_validations/playbook.py

```diff
@@ -258,6 +263,44 @@ def is_valid_deprecated_playbook(self) -> bool:
                     is_valid = False
         return is_valid
 
+    def is_valid_playbook_description(self) -> bool:
+        """Check whether the playbook description contains Escape Sequences
+
+        Returns:
+            bool. True if the playbook description is valid - not contains Escape Sequences, False otherwise.
+        """
+        is_valid = True
+        description = self.current_file.get('description', '')
+        if is_description_contains_escape_sequences(description):
+            error_message, error_code = Errors.invalid_playbook_description()
+            if self.handle_error(error_message, error_code, file_path=self.file_path):
+                is_valid = False
+        return is_valid
+
+    def is_valid_tasks_description(self) -> bool:
+        """Check whether the playbook tasks description contains Escape Sequences
+
+        Returns:
+            bool. True if the playbook tasks description is valid - not contains Escape Sequences, False otherwise.
+        """
+        is_valid = True
+        invalid_tasks_list = []
+        tasks = self.current_file.get('tasks', {})
+
+        for id, task in tasks.items():
+            pb_id = task.get('id', '')
```

Comment:
Maybe this should be id and name so it looks better - 
like here:
![image](https://user-images.githubusercontent.com/50295826/99416861-f52ceb80-2901-11eb-920b-ac43cdfa7bf3.png)
it will be:
`1 - Arcsight - Get Case info`

can you try it and see if it looks better?

Reply:
done